lib/PuppeteerSharp/PuppeteerSharp.csproj

```diff
@@ -136,6 +136,9 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DocumentationFile>bin\Debug\netstandard2.0\PuppeteerSharp.xml</DocumentationFile>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">

```

Comment:
can't we just remove the condition and have a single `DocumentationFile` property?

Reply:
Done